tests/src/python/test_provider_postgres.py

```diff
@@ -2483,6 +2485,8 @@ def testFeatureCountEstimatedOnView(self):
             'test', 'postgres')
         self.assertTrue(vl.isValid())
         self.assertTrue(vl.featureCount() > 0)
+        vl.setSubsetString('"pk" = 3')
+        self.assertEqual(vl.featureCount(), 1)
```

Comment:
This is an estimation so we cannot be sure that the count would be one. Maybe we should just check that it is greater than 0

Reply:
The test is on the primary key, so I assmued that one pk value should always filter to one feature only, and the estimation is correct with the current test dataset.

But I aggree that if this test can't be trusted enough, we should simply remove this assertion because testing only if > 0 is the same than testing without subsetString